orttraining/tools/scripts/pipeline_model_split.py

```diff
@@ -0,0 +1,376 @@
+import sys
```

Comment:
nit: please consider use autoPEP to automatically format the code. If you use VSCode, there should be plugins and just need to add one line to settings.json:
"editor.formatOnSave": true,